halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/GateService.java

```diff
@@ -84,8 +86,12 @@ public Type getType() {
 
   private GateProfileFactory getGateProfileFactory(String deploymentName) {
     String version = artifactService.getArtifactVersion(deploymentName, SpinnakerArtifact.GATE);
-    if (Versions.lessThan(version, BOOT_UPGRADED_VERSION)) {
-      return boot128ProfileFactory;
+    try {
```

Comment:
This isn't here to handle the `local:` prefix, it's here so that my gate image tag doesn't have to follow semvar.